src/Components/test/E2ETest/Infrastructure/SeleniumStandaloneServer.cs

```diff
@@ -40,15 +43,30 @@ private SeleniumStandaloneServer()
             var port = FindAvailablePort();
             Uri = new UriBuilder("http", "localhost", port, "/wd/hub").Uri;
 
-            var process = Process.Start(new ProcessStartInfo
+            var psi = new ProcessStartInfo
```

Comment:
Loosely based on what we do in Razor integration tests. The current fixture failure doesn't include the output from npm which goes a fairly long way in diagnosing the reason the server did not start. In my case it was because setting `JAVA_HOME` wasn't enough and following the instructions to launch selenium server standalone wasn't the right thing to do.